src/SnetApp.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import SandboxApp  from "./sandbox/SandboxApp";
+import App from "./App";
+
+export default () => {
```

Comment:
missing Lazy loading here

Reply:
Have lazy loading for service details. Don't think we need it here.